.github/workflows/cypress.yml.disabled

```diff
@@ -6,134 +6,42 @@ on:
-
-      - name: Get bot token from Parameter Store
-        uses: marvinpinto/action-inject-ssm-secrets@v1.2.1
-        with:
-          ssm_parameter: /devops/VA_VSP_BOT_GITHUB_TOKEN
-          env_variable_name: VA_VSP_BOT_GITHUB_TOKEN
-
-      - name: Get latest GHA Runner AMI ID
-        run: |
-          echo "RUNNER_AMI_ID=$(aws ec2 describe-images \
-          --owners 008577686731 \
-          --filters "Name=state,Values=available" "Name=name,Values=platform-gha-runner-ubuntu*" \
-          --query 'sort_by(Images,&CreationDate)[-1].ImageId' \
-          --output text)" >> $GITHUB_ENV
-
-      - name: Get Subnet with the most free IPs # We will run these in the dsva-vagov-utility-2x subnet, so filter for those
-        run: |
-          echo "SUBNET_ID=$(aws ec2 describe-subnets \
-          --filters "Name=tag:Name,Values=dsva-vagov-utility-subnet-2*" \
-          --query 'sort_by(Subnets,&AvailableIpAddressCount)[-1].SubnetId' \
-          --output text)" >> $GITHUB_ENV
-
-      - name: Start EC2 Runner
-        id: start-ec2-runner
-        uses: department-of-veterans-affairs/ec2-github-runner@main
-        with:
-          mode: start
-          github-token: ${{ env.VA_VSP_BOT_GITHUB_TOKEN }}
-          ec2-image-id: ${{ env.RUNNER_AMI_ID }}
-          ec2-instance-type: ${{ env.INSTANCE_TYPE }}
-          subnet-id: ${{ env.SUBNET_ID }}
-          security-group-id: sg-0e23b56be3798e3a1
-          max_attempts: 3
-          aws-resource-tags: >
-            [
-              {"Key": "Name", "Value": "dsva-vagov-content-build-gha-runner"},
-              {"Key": "project", "Value": "vagov"},
-              {"Key": "office", "Value": "dsva"},
-              {"Key": "application", "Value": "gha-runner"},
-              {"Key": "VAECID", "Value": "AWG20180517003"},
-              {"Key": "environment", "Value": "utility"}
-            ]
-
   build:
     name: Build static
-    runs-on: ${{ needs.start-runner.outputs.label }}
-    needs: start-runner
-    timeout-minutes: 5
+    runs-on: [self-hosted, asg]
+    timeout-minutes: 15
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version-file: .nvmrc
 
       - name: Setup Yarn
-        run: npm i -g yarn@1.22.19
-
-      - name: Install Npm Dependencies
         run: |
-          yarn set version stable
-          HUSKY=0 yarn install --immutable
-
-      - name: check node env
-        id: check-node-env
-        run: echo $NODE_ENV
+         corepack enable
+         corepack hydrate ./.yarn/releases/corepack.tgz
+         corepack prepare yarn@3.6.1 --activate
+
+      # todo: figure out where/how to cache this
+      # necessary to do this way because cypress postinstall command ignores ssl/certs settings
+      - name: Download Cypress Binary
+        run: curl https://download.cypress.io/desktop/13.2.0?platform=linux&arch=x64 > /home/runner/runner1/_work/next-build/next-build/.yarn/cypress/cypress.zip
```

Comment:
As long as we're doing this, the destination could be `$CYPRESS_CACHE_FOLDER` I think. But, see above note re: `runner1` not being a given.